bindinfo/handle.go

```diff
@@ -24,6 +24,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	tablefilter "github.com/pingcap/tidb-tools/pkg/table-filter"
```

Comment:
I take a look at the README of this package and find it supports many features, such as allow/block list and file import. Is that expected?

Reply:
You are right. I've refined related logic and added more tests, PTAL @time-and-fate